src/Microsoft.AspNetCore.Authentication.Facebook/Properties/Resources.Designer.cs

```diff
@@ -15,16 +15,14 @@ internal static class Resources
         /// </summary>
         internal static string Exception_OptionMustBeProvided
         {
-            get { return GetString("Exception_OptionMustBeProvided"); }
+            get => GetString("Exception_OptionMustBeProvided");
```

Comment:
revert? It's kindof weird to include these with your change. Especially when we want to backport it.

Reply:
I guess, we should fix this repo to be clean when you run ./build.cmd /t:resx which is what I did to generate the new resource

We aren't planning on porting this one back are we?  The other fix is important, this one just results in a better error message

Reply:
I pulled the other resx changes into: https://github.com/aspnet/Security/pull/1424

Reply:
I'll get that in first, and then rebase so these resx diffs will go away